front/app/modules/commercial/representativeness/admin/containers/Dashboard/fakeData.ts

```diff
@@ -0,0 +1,159 @@
+import moment from 'moment';
+
```

Comment:
Dummy data used to render the dummy charts right now. Will be removed in the future